### ISSUE TITLE
Allow setting Payload::setWrapper from the commandline

### ISSUE THREAD

**Issue by beberlei** [phpbench/phpbench#410]
To wrap the benchmarking with an external profiler such as Tideways, we need to call `Payload::setWrapper()`, but that method is not changeable without adding a new extension.

Would `phpbench --wrapper "foobar" myscript.php` be possible?

**Comment by dantleech** [comment]
Yeah I think so, it might then also make sense to be able to define the PHP binary and INI parameters both in config and from the CLI.

```
$ phpbench run --php-binary=php7 --php-ini-params='foo: bar, bar: foo' --php-wrapper=tideways
```

Separately, do you think it would be worth creating a Tideways extension? It could f.e. return the URL to the profile after execution and limit the iterations to 1.

**Comment by beberlei** [comment]
@dantleech Yes that makes sense to have, I will see when we have time working on this.

**Comment by dantleech** [comment]
Fixed in #413
